src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionCategoryServiceImplTest.java

```diff
@@ -35,8 +36,14 @@
 @ExtendWith(MockitoExtension.class)
 public class FinancialTransactionCategoryServiceImplTest {
 
-    public static final long ID_1L = 1L;
-
+    private static final long FINANCIAL_TRANSACTION_CATEGORY_ID_1L = 1L;
+    private static final long FINANCIAL_TRANSACTION_CATEGORY_ID_2L = 2L;
```

Comment:
Code formatting.
Please add one blank line before each class field.

Reply:
Done.

Reply:
In the end it should looks like this:
```java
@ExtendWith(MockitoExtension.class)
public class FinancialTransactionCategoryServiceImplTest {

    private static final long FINANCIAL_TRANSACTION_CATEGORY_ID_1L = 1L;
	
    private static final long FINANCIAL_TRANSACTION_CATEGORY_ID_2L = 2L;
	
    private static final long FINANCIAL_TRANSACTION_CATEGORY_ID_3L = 3L;
	
    private static final long USER_ID_1L = 1L;
	
    private static final String FINANCIAL_TRANSACTION_CATEGORY_NAME_FIRST = "First";
	
    private static final String FINANCIAL_TRANSACTION_CATEGORY_NAME_SECOND = "Second";
	
    private static final String FINANCIAL_TRANSACTION_CATEGORY_NAME_THIRD = "Third";
	
    private static final String FINANCIAL_TRANSACTION_CATEGORY_NAME_EXAMPLE_NAME = "example name";
	
    public static final String CATEGORY_NAME = "Name";
```

Reply:
Done.